src/dosext/dpmi/dpmi.c

```diff
@@ -1113,8 +1113,9 @@ int SetDescriptor(unsigned short selector, unsigned int *lp)
   return ret;
 }
 
-void GetFreeMemoryInformation(unsigned int *lp)
+void GetFreeMemoryInformation(unsigned int *addr, dosaddr_t daddr)
```

Comment:
No-no, please use local buffers at a
caller site for now.
Or maybe just proceed to adding the
SEL_ADR_W()?